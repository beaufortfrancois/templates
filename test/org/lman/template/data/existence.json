{
  "null": null,
  "booleans": {
    "true": true,
    "false": false
  },
  "numbers": {
    "zero": 0,
    "one": 1
  },
  "strings": {
    "empty": "",
    "nonempty": "foo"
  },
  "arrays": {
    "empty": [],
    "nonempty": [1,2,3]
  },
  "objects": {
    "empty": {},
    "nonempty": {"foo": "bar"}
  }
}
