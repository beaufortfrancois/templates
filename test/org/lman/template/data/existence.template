null:
{{?null}}null{{/}}

booleans:
{{?booleans.true}}boolean.true{{/}}
{{?booleans.false}}boolean.false{{/}}

numbers:
{{?numbers.zero}}numbers.zero{{/}}
{{?numbers.one}}numbers.one{{/}}

strings:
{{?strings.empty}}strings.empty{{/}}
{{?strings.nonempty}}strings.nonempty{{/}}

arrays:
{{?arrays.empty}}arrays.empty{{/}}
{{?arrays.nonempty}}arrays.nonempty{{/}}

objects:
{{?objects.empty}}objects.empty{{/}}
{{?objects.nonempty}}objects.nonempty{{/}}
