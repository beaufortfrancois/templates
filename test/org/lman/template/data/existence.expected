null:

booleans:
boolean.true

numbers:
numbers.zero
numbers.one

strings:
strings.empty
strings.nonempty

arrays:
arrays.nonempty

objects:
objects.empty
objects.nonempty
